scripts/benchmarks/format-benchmark-record.js

```diff
@@ -127,6 +133,34 @@ const formatDecileGenerator = function(record) {
   };
 };
 
+// Looks in measures-data.json for an existing measureIds and
+// walks through the original, capitalized + removed spaces,
+// and capitalized + underscored spaces versions.
+// If found, returns the measureId from the measures-data.json file.
+// If none are found, return the padded number or non-spaced version
+const formatMeasureId = (measureId) => {
+  const measureIdRemovedSpaces = measureId.replace(/\s+/g, '');
+  const measureIdUnderscore = measureId.replace(/\s+/g, '_');
+  const measureIdCandidates = [
+    measureId,
+    measureIdRemovedSpaces,
+    measureIdUnderscore
+  ];
+
+  for (const formattedMeasureID of measureIdCandidates) {
+    const measure = MEASURE_ID_TO_MEASURE_MAP[formattedMeasureID];
+    if (measure) return measure.measureId;
+  };
+
+  // If all digits, pad with zeros up to the thousands
+  // else, return a nonspaced version
+  if (measureId.match(/^\d+$/)) {
+    return ('0000' + measureId).slice(-4);
```

Comment:
What's the reason for padding in this case?

Reply:
It's the format numeric-only benchmarks are using right now. There isn't a set standard but I wanted to conform to something we have already.

Reply:
They're actually padded to the hundredth place in `benchmarks/2017.json`. Made the change and added a test.

Reply:
👍